birdhouse/pavics-compose.sh

```diff
@@ -90,7 +94,7 @@ if [ -z "$PAVICS_FQDN_PUBLIC" ]; then
   export PAVICS_FQDN_PUBLIC="$PAVICS_FQDN"
 fi
 
-if [ "$ALLOW_UNSECURE_HTTP" = "True" ]; then
+if [ x"$ALLOW_UNSECURE_HTTP" = x"True" ]; then
```

Comment:
I think the 'x' in this situation only makes sense if the double quotes are not present to avoid the syntax error on missing variable expansion. With the double quotes it expands as `"" = "True"` which should behave correctly.

Reply:
I've read it also helps to prevent code injection in shell script.  The 'x' in front basically change the command if someone put the command there instead of the expected value.  Agreed this is a bit on a paranoid side, but having the 'x' there does not harm the functionality.